wallet/how-to/use-non-evm-networks/starknet/manage-starknet-accounts.md

```diff
@@ -0,0 +1,155 @@
+---
+description: Manage Starknet accounts in MetaMask.
+sidebar_position: 2
+---
+
+# Manage Starknet accounts
+
+## Account creation
+
+Account creation in Starknet is handled by the wallet provider. As a dapp developer, you do not create accounts directly. Instead, you guide users to create an account with their preferred wallet provider.
+
+:::note
+
+Currently, multiple Starknet accounts are not supported in the Starknet Snap.
+
+:::
+
+## View account information
+
+After a user connects, you can display the account details, such as the account address: 
+
+```javascript
```

Comment:
Please update this and all other code samples to use double quotes instead of single quotes.